configure.ac

```diff
@@ -859,6 +866,19 @@ else
   fi
 fi
 
+
+
+dnl Use rdseed to gather entropy
+if test "x$use_rdseed" = "xyes"; then
+        ARM_CC=`which arm-linux-gnueabihf-g++`
```

Comment:
I think this is incorrect. The fact that I have an ARM compiler installed does not mean I am compiling for ARM. What was not functioning with $host detection?

Reply:
I tried several versions of the ``uname`` command which tells architecture, but none of them worked. Since I don't have a ARM based system, I rented an AWS ARM Linux machine and this was the only trial that worked both on that one and on the CI pipeline here. As for the ``$host`` variable, it is not defined. I am not quite sure what you mean by it.

Reply:
@patricklodder Any suggestions on how to make this work would be greatly appreciated.

Reply:
Fixed it for you in https://github.com/ReverseControl/dogecoin-1/pull/15 using `$host`

Reply:
Thanks. Merged.